src/Components/QuestionList/QuestionFilter/DifficultySelect/DifficultySelect.tsx

```diff
@@ -9,27 +12,38 @@ interface DifficultyProps {
 }
 
 export const DifficultySelect = ({ value, setValue }: DifficultyProps) => {
+    const [radioValue, setRadioValue] = useState("1");
+
+    const Difficulties = Object.values(Difficulty);
+    Difficulties.splice(Difficulties.length - 1, 1);
```

Comment:
this is bad, we don't want to manipulate data on component render. It can get manipulated every render

Reply:
If you want to filter out a specific value, then call that out with filter